core/prisma/schema/schema.prisma

```diff
@@ -579,6 +580,9 @@ model ApiAccessToken {
   PubValueHistory PubValueHistory[]
   InviteHistory   InviteHistory[]
 
+  // has unique index with isSiteBuilderToken = true and communityId
+  isSiteBuilderToken Boolean @default(false)
+
```

Comment:
i know eric things rando boolean columns are cringe, but i couldnt think of anything better!